android/src/amazon/java/com/dooboolab/RNIap/RNIapAmazonListener.kt

```diff
@@ -62,6 +68,27 @@ class RNIapAmazonListener(private val reactContext: ReactContext) : PurchasingLi
                     item.putString("productId", product.sku)
                     item.putString("price", priceNumber.toString())
                     item.putString("type", productTypeString)
+                    // Begin Add localizedPrice12
+                    val localizedPrice12 = if (priceNumber.toDouble().compareTo(0.00) > 0) {
+                        val price12 = priceNumber.toDouble().toBigDecimal().multiply(BigDecimal.valueOf(2))
+                        val currency = try {
+                            Currency.getInstance("VND")
```

Comment:
Should this be a parameter? Or maybe it can be obtained from the payload? This makes it very specific to your locale